src/button-dropdown/styles.scss

```diff
@@ -58,7 +58,8 @@ $dropdown-trigger-icon-offset: 2px;
   display: flex;
 
   & > .trigger-item {
-    & > button:focus {
```

Comment:
In some permutations, the trigger is actually not a button but a hyperlink. So it's better to match by the class name rather than the element type.